inc/customizer/extend-custom-controls/src/color-group/color-group-component.js

```diff
@@ -10,13 +14,68 @@ const ColorGroupComponent = props => {
 		name
 	} = props.control.params;
 
+	const linked_sub_colors = AstraBuilderCustomizerData.js_configs.sub_controls[name];
+	const color_group = [],
+		tooltips = [];
+
+	Object.entries( linked_sub_colors ).map( ( [ key,value ] ) => {
+		color_group[value.name] = wp.customize.control( value.name ).setting.get();
+		tooltips[value.name] = value.title;
+	});
+
+	const[ state , setState ] = useState(color_group);
+
+	const handleChangeComplete = ( key, color='' ) => {
+
+		let updateState = {
+			...state
+		};
+
+		let value;
+
+		if (typeof color === 'string' || color instanceof String) {
+			value = color;
+		} else if (undefined !== color.rgb && undefined !== color.rgb.a && 1 !== color.rgb.a) {
+			value = 'rgba(' + color.rgb.r + ',' + color.rgb.g + ',' + color.rgb.b + ',' + color.rgb.a + ')';
```

Comment:
```suggestion
			value = `rgba(${color.rgb.r},${color.rgb.g},${color.rgb.b},${color.rgb.a})`;
```

Tip - [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) can be used to build strings.

@vrundakansara - Do confirm if we should add this only in one place now or we should do one commit in all the places?

Reply:
Done here @Nikschavan sir in all cases - https://github.com/brainstormforce/astra/pull/2292/commits/5c9fd9bd64bb769ac4c4a2b2385921841c84fdbc

Please review. 

cc: @vrundakansara